UPGRADE.md

```diff
@@ -3,6 +3,22 @@
 This document serves as a reference to upgrade your current BetterReflection installation if improvements, deprecations
 or backwards compatibility (BC) breakages occur.
 
+## 5.0.0
+
+### BC Breaks
+* Only PHP 8.0 is supported.
+* Adapters don't have `::export()` method anymore because these methods were removed from PHP. These methods has been removed:
+  * `\Roave\BetterReflection\Reflection\Adapter\ReflectionClass::export()` 
+  * `\Roave\BetterReflection\Reflection\Adapter\ReflectionFunction::export()` 
+  * `\Roave\BetterReflection\Reflection\Adapter\ReflectionMethod::export()` 
+  * `\Roave\BetterReflection\Reflection\Adapter\ReflectionObject::export()` 
+  * `\Roave\BetterReflection\Reflection\Adapter\ReflectionParameter::export()` 
+  * `\Roave\BetterReflection\Reflection\Adapter\ReflectionProperty::export()` 
+* Casting to string is compatible with PHP: 
```

Comment:
This `UPGRADE.md` line doesn't make much sense to the reader: can we rephrase it?

Reply:
My english is poor so feel free to propose better text please :)